src/com/xilinx/rapidwright/debug/ProbeRouter.java

```diff
@@ -80,7 +80,7 @@ public static void updateProbeConnections(Design d, Map<String,String> probeToTa
         ArrayList<SitePinInst> pinsToRoute = new ArrayList<>();
         for (Entry<String,String> e : probeToTargetNets.entrySet()) {
             String hierPinName = e.getKey();
-            String cellInstName = EDIFTools.getHierarchicalRootFromPinName(hierPinName);
+            String cellInstName = d.getNetlist().getHierCellInstFromName(hierPinName).getParent().toString();
             EDIFCellInst i = d.getNetlist().getCellInstFromHierName(cellInstName);
```

Comment:
Seems a bit awkward to get the `EDIFHierCellInst`'s parent name and use that to lookup its `EDIFCellInst`?

Reply:
Good point, I have simplified it a bit.